util/logger/logger.go

```diff
@@ -89,14 +92,37 @@ func addFields(event *zerolog.Event, keyvals ...interface{}) *zerolog.Event {
 }
 
 func NewSubLogger(name string, obj fmt.Stringer) *SubLogger {
+	var writers []io.Writer
+
+	if getLoggersInst().config.ConsoleLoggingEnabled {
+		writers = append(writers, zerolog.ConsoleWriter{Out: os.Stderr})
```

Comment:
Define two writers:
1- Console Writer
2- File Writer


File writer name is "pactus.log", the MaxSize is 100MB, Directory for backup is "logs" and out put is JSON (I am assuming that the default is JSON)